prime-dotnet-webapi/Configuration/Database/Seeded Tables/EmailTemplateConfiguration.cs

```diff
@@ -12,22 +12,23 @@ public override IEnumerable<EmailTemplate> SeedData
+                    new EmailTemplate { Id = 13, EmailType = EmailTemplateType.SiteRegistrationSubmission,             Template = "<p> A new PharmaNet site registration has been received. See the attached registration and organization agreement for more information. @if (!string.IsNullOrWhiteSpace(Model.Url)) { @(\"To access the Business Licence, click this\") <a href=\"@Model.Url\" target=\"_blank\">link</a>@(\".\") } </p>", ModifiedDate = SEEDING_DATE, CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE},
+                    new EmailTemplate { Id = 14, EmailType = EmailTemplateType.OrganizationClaimApprovalNotification,  Template = "Your claim of the organization @Model.OrganizationName, of which the site with site ID/PEC @Model.ProvidedSiteId is part of, has been approved.  You now have access to site registration for this organization.", ModifiedDate = SEEDING_DATE, CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE},
+                    new EmailTemplate { Id = 15, EmailType = EmailTemplateType.SiteReviewedNotification,               Template = "A PRIME Admin has reviewed the site registration for PEC/SiteID# @Model.Pec. @(!string.IsNullOrWhiteSpace(Model.Note) ? $\"The following notes were added: {Model.Note}\" : \"\")", ModifiedDate = SEEDING_DATE, CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE},
+                    new EmailTemplate { Id = 16, EmailType = EmailTemplateType.SiteActiveBeforeRegistrationSubmission, Template = "Thank you for registering your site (SiteID: @Model.Pec) in PRIME. If you need to update any site information in PRIME, you may log in at any time using your mobile BC Services Card. If you have any questions, please phone 1 - 844 - 397 - 7463 or email <a href=\"mailto:PRIMEsupport@gov.bc.ca\" target=\"_top\">PRIMEsupport@gov.bc.ca</a>. Thank you.", ModifiedDate = SEEDING_DATE, CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE},
+                    new EmailTemplate { Id = 17, EmailType = EmailTemplateType.PaperEnrolleeSubmission               , Template = "Your request for PharmaNet access has been approved and recorded in PRIME. When it is possible for you to do so, you must enrol in PRIME using your mobile BC Services Card. <br> <br> <strong> Your temporary GPID is @Model.GPID. </strong> <br> <br> The first time you log into PRIME you should be asked if you have previously received permission to access PharmaNet via an offline process. If you do not see this prompt, please stop your enrollment and contact <a href=\"mailto:PRIMEsupport@gov.bc.ca\" target=\"_top\">PRIMEsupport@gov.bc.ca</a>", ModifiedDate = SEEDING_DATE, CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE}
```

Comment:
Minor formatting wise pull the semi-colon back, but keeps the spaces

![image](https://user-images.githubusercontent.com/599498/141837073-51635492-a5de-4c87-8d6d-a7f8593e2c57.png)

Reply:
Fixed